cds_ils/migrator/eitems/api.py

```diff
@@ -94,7 +94,20 @@ def add_eitem_extra_metadata(eitem, document):
     PROVIDERS_MAPPING = {
         "safari": "SAF",
         "springer": "SPR",
-        "ebl": "EBL"
+        "ebl": "EBL",
```

Comment:
please remove this part, as it was meant for migration, and these providers do not appear in the migrated data